### DISCUSSION TITLE
HW 3 P1 Q1

### DISCUSSION THREAD

**Discussion by awc17002** [cooperrc/computational-mechanics#108] [General]
When going through the reading, we have to click to track the location of the ball so we can use the coordinates in the first part of the homework. My problem is despite being very careful, when I plot the mouse clicks, it still shows I double clicked. Has anyone else run into that problem and if so, how they got around it. I've attached a screenshot of the graph to show what I'm talking about. 
![Comp_Mech_Capture](https://user-images.githubusercontent.com/112645322/193868739-e8bae9b9-6d1a-4f7b-8c80-f2c5597d2165.PNG)

**Comment by cooperrc** [comment]
Hmm, that's strange. I don't know why you're registering x2 clicks on _all_ of the points. 

Here's a quick work-around for the given data, lets say you have `y`:

```python
y = np.array([
-0.205,
-0.204,
-0.201,
-0.200,
-0.195,
-0.196,
...
-0.506
-0.504])
```

Grab every other y-value with

```python
y[::2]
out:
np.array([
-0.205,
-0.201,
-0.195
...
-0.506])

```

then, you can get time, `t`, into the correct for again with

```python
t = np.arange(0,len(y))/60
```

because frames happen every 1/60 secs
